scipy/stats/stats.py

```diff
@@ -2858,13 +2889,134 @@ def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
         med = np.apply_over_axes(center, arr, axis)
         mad = np.median(np.abs(arr - med), axis=axis)
 
-    return scale * mad
+    return mad / scale
+
+
+# Keep the top newline so that the message does not show up on the stats page
+_median_absolute_deviation_deprec_msg = """
+To preserve the existing behavior, use
+`scipy.stats.median_abs_deviation(..., scale=1.4826)`.
```

Comment:
I think this message should now show `scale=1/1.4826`.

Here's scipy 1.4.1:
```
In [77]: stats.median_absolute_deviation([1, 2, 4, 5, 9])
Out[77]: 2.9652
```
To get that same behavior with `median_abs_deviation`, I need to use `1/1.4826`:
```
In [17]: stats.median_abs_deviation([1, 2, 4, 5, 9], scale=1/1.4826)
Out[17]: 2.9652
```

Reply:
And perhaps further clarify the intent of this message by add "default" in the previous line, so it says
```
To preserve the existing default behavior, use
```

Reply:
yes, done on both counts